.github/workflows/testing.yml

```diff
@@ -33,6 +34,9 @@ jobs:
         run: pip uninstall -y h5py
       - name: Install ROS3
         run: conda install -c conda-forge h5py
+      - name: temporary-pynwb-pin
+        run: pip install pynwb==2.1.0
```

Comment:
No need for a `requirements-testing.txt` or `extra_requires` since this is  the only thing needed to pin for the tests (temporarily, until #288)